modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommandList.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ignite.internal.commandline.management;
+
+import org.apache.ignite.internal.commandline.argument.CommandArg;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Set of management commands.
```

Comment:
Lets describe in javadocs command syntax and usages.

Reply:
@akuznetsov-gridgain No, we did this in visor cmd and it's totally uncomfortable when you modify command you need change: help command output, javadoc. Also other commands doesn't do this